### PR TITLE
Connected events and news to the backend

### PR DESCRIPTION
**Pull request by florisdobber**
Changed the endpoints of events and news for all modes to the new backend, then made sure the news and events pages still work as expected.

### CODE REVIEW

**Comment by florisdobber** [comment]
Trello tickets: 
[https://trello.com/c/W9wugtXd ](url)
[https://trello.com/c/LkIhylT5](url)

File: src/views/Events.vue [hunk 1]
```diff
@@ -109,13 +109,14 @@ export default {
...
+    //legacy code
     unixToDate(unix) {
-      return unix * 1000;
+      return unix;
     },
 
     //Helper function: determines if a given date is more than 24 hours in the past
     isPast(date) {
-      return this.unixToDate(date) <= Date.now() - 24 * 60 * 60 * 1000; //day in milliseconds
+      return new Date(date) <= Date.now();
```
Comment (liammoyn) [inline src/views/Events.vue]: Update the comment to reflect what this method is doing

File: src/views/Events.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <list-card
           v-for="event in futureEvents"
           :key="event.id"
-          :title="event.title"
+          :title="event.name"
```
Comment (liammoyn) [inline src/views/Events.vue]: If you're going to change the event to name then you should probably change this attribute name as well

File: src/store/index.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default new Vuex.Store({
       return state.posts;
     },
     GET_POST_BY_ID: state => id => {
-      return state.posts.find(post => post.id === id);
+      return state.posts.find(post => post.id == id);
```
Comment (liammoyn) [inline src/store/index.js]: Why is this necessary?

File: src/views/Events.vue [hunk 4]
```diff
@@ -109,13 +109,14 @@ export default {
     },
 
     //unixToDate: converts the given unix timestamp to a javascript date number
+    //legacy code
```
Comment (liammoyn) [inline src/views/Events.vue]: There's really zero reason to have "legacy code" in a code base that is currently under development. If this method does nothing then just remove it

File: src/store/index.js (hunk 3 above)
Comment (jackblanc) [reply src/store/index.js]: type checking string vs number

File: src/views/Events.vue (hunk 1 above)
Comment (florisdobber) [reply src/views/Events.vue]: Method still does the same thing, only adjusted for the different format.

File: src/components/ListCard.vue [hunk 5]
```diff
@@ -5,7 +5,7 @@
     </div>
     <div>
       <p class="post-date">{{ dateString }}</p>
-      <p class="big align-left">{{ title }}</p>
+      <p class="big align-left">{{ name }}</p>
```
Comment (jackblanc) [inline src/components/ListCard.vue]: change this back to title

it breaks news
title is generalizable, name is not.

Sorry for the confusion here.
